src/kernel_abi.h

```diff
@@ -1781,6 +1781,29 @@ struct BaseArch : public wordsize,
       aligned_u64 line_info;
       __u32 line_info_cnt;
     };
+
+    struct { /* anonymous struct used by BPF_PROG_QUERY command */
+      union {
+        __u32   target_fd;      /* target object to query or ... */
+        __u32   target_ifindex; /* target ifindex */
```

Comment:
Get rid of the extra spaces between the type and the field name, here and below.